google-cloud-bigquery/acceptance/bigquery/table_test.rb

```diff
@@ -156,6 +156,34 @@
     stale.etag.must_equal fresh.etag
   end
 
+  it "create table with cmek sets encryption_configuration" do
+    begin
+      encrypt_config = Google::Cloud::Bigquery::EncryptionConfiguration.new
```

Comment:
I would like to see an `encryption_configuration` helper method so users don't have to type `Google::Cloud::Bigquery::EncryptionConfiguration.new`.

Reply:
Sorry, I don't know what you mean. Is there an example?

Reply:
Instead of:

```ruby
encrypt_config = Google::Cloud::Bigquery::EncryptionConfiguration.new
encrypt_config.kms_key_name =  "projects/cloud-samples-tests/locations/us-central1" +
                               "/keyRings/test/cryptoKeys/test"
```

It would be more convenient to type:

```ruby
encrypt_config = bigquery.encryption_configuration \
  kms_key_name: "projects/cloud-samples-tests/locations/us-central1" +
                "/keyRings/test/cryptoKeys/test"
```

I think it would also be useful to have a helper/convenience method that would create the `kms_key_name` value. A helper method that took the arguments and created the full resource string.